client/state/action-types.js

```diff
@@ -669,6 +669,9 @@ export const SITE_VOUCHERS_REQUEST_SUCCESS = 'SITE_VOUCHERS_REQUEST_SUCCESS';
 export const SITE_WORDPRESS_UPDATE_REQUEST = 'SITE_WORDPRESS_UPDATE_REQUEST';
 export const SITE_WORDPRESS_UPDATE_REQUEST_FAILURE = 'SITE_WORDPRESS_UPDATE_REQUEST_FAILURE';
 export const SITE_WORDPRESS_UPDATE_REQUEST_SUCCESS = 'SITE_WORDPRESS_UPDATE_REQUEST_SUCCESS';
+export const SOCIAL_LOGIN_REQUEST = 'SOCIAL_LOGIN_REQUEST';
```

Comment:
Do we plan to consume those action types? :) I assume, we do. I think that at least `SOCIAL_LOGIN_REQUEST_FAILURE` is going to be useful to display error message when something goes wrong.